patches/typescript-react-function-component-props-handler+1.1.1.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/typescript-react-function-component-props-handler/index.js b/node_modules/typescript-react-function-component-props-handler/index.js
+index 5c636b9..2f3cb68 100644
+--- a/node_modules/typescript-react-function-component-props-handler/index.js
++++ b/node_modules/typescript-react-function-component-props-handler/index.js
+@@ -47,9 +47,9 @@ function checkForProptypes(path, paramTypeName) {
+ }
+ 
+ function setParamsTypeDefinitionFromFunctionType(documentation, path) {
+-    if (path.parentPath.node.init && path.parentPath.node.init.params.length === 0) {
++    if (path.parentPath.node.init  && Array.isArray(path.parentPath.node.init.params) && path.parentPath.node.init.params.length === 0) {
```

Comment:
I'm missing a bit of context here. Can you explain how this makes `forwardRef` work?
Thanks 🤗

Reply:
I see, thanks.

Btw, this package does not seem maintaned since a couple of years.

We can either:
* keep using the patch.
* copy the implementation file and use it in our repo, without relying on the package.